dask/array/core.py

```diff
@@ -1157,11 +1157,13 @@ def __repr__(self):
         """
         chunksize = str(self.chunksize)
         name = self.name.rsplit("-", 1)[0]
-        return "dask.array<%s, shape=%s, dtype=%s, chunksize=%s>" % (
+        return "dask.array<%s, shape=%s, dtype=%s, chunksize=%s, meta=%s.%s>" % (
```

Comment:
Should we use the term `meta` here, or some other term?  Meta seems like a term for developers, but users might prefer something like "chunk type" to go along with chunk size just before.

Reply:
Hmm that's a fair point. I agree that `meta` may be confusing to users, especially if they don't ever interact with a `meta=` parameter directly. Using `chunktype=` seems like a reasonable thing to do. FWIW that would also more consistent with the HTML repr.

Reply:
I have a slight preference for "chunktype" (for consistency with "chunksize") instead of "chunk type"

Reply:
Sounds good to me.  Sorry for not mentioning this before you did all of the work in the latest two commits.  I should have thought a bit more before when the issue was raised.

Reply:
Not a problem at all. I was thinking more from a developer's perspective, not a user. Thank you for bringing it up : )